src/migrate.py

```diff
@@ -70,7 +70,8 @@
     pts = total_points[user["id"]] if user["id"] in total_points else 0
     ccnt = contest_cnt[user["id"]] if user["id"] in contest_cnt else 0
     pcnt = problem_cnt[user["id"]] if user["id"] in problem_cnt else 0
+    email = email.lower() #  for the lowercasing email PR
```

Comment:
I don't think email here is defined, go for `user["email"].lower()`

Reply:
i fixed it while you were reviewing by looking at the build log :skull: